amgut/handlers/auth_handlers.py

```diff
@@ -23,6 +23,8 @@ def get(self):
     @authenticated
     def post(self):
         skid = self.current_user
+        # log user out for register process
+        self.clear_cookie("skid")
         tl = text_locale['handlers']
         info = {}
         for info_column in ("email", "participantname", "address", "city",
```

Comment:
This is a loop to go through and get the form values from the registration page. We have checks in place on that page to make sure they aren't none.